Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -160,266 +301,291 @@ def fetch_incidents(client: MsGraphClient, fetch_time: str, fetch_limit: int, fi
-        'ID': alert_details['id'],
-        'Title': alert_details['title'],
-        'Category': alert_details['category'],
-        'Severity': alert_details['severity'],
-        'CreatedDate': alert_details['createdDateTime'],
-        'EventDate': alert_details['eventDateTime'],
-        'Status': alert_details['status'],
-        'Vendor': alert_details['vendorInformation']['vendor'],
-        'Provider': alert_details['vendorInformation']['provider']
-    }
-    ec = {
-        'MsGraph.Alert(val.ID && val.ID === obj.ID)': context
-    }
+                vendor_info_title = 'Vendor Information for Alert'
+                hr += tableToMarkdown(vendor_info_title, vendor_info_hr, removeNull=True)
+
+        if 'VulnerabilityStates' in fields_list or show_all_fields:
+            vulnerability_states = alert_details['vulnerabilityStates']
+            if vulnerability_states:
+                vulnerability_states_hr = []
+                for state in vulnerability_states:
+                    vulnerability_states_hr.append({
+                        'CVE': state['cve'],
+                        'Severity': state['severity'],
+                        'WasRunning': state['wasRunning']
+                    })
+                vulnerability_states_title = 'Vulnerability States for Alert'
+                hr += tableToMarkdown(vulnerability_states_title, vulnerability_states_hr, removeNull=True)
+
+        if 'RegistryKeys' in fields_list or show_all_fields:
+            registry_keys = alert_details['registryKeyStates']
+            if registry_keys:
+                registry_keys_hr = []
+                for r_key in registry_keys:
+                    r_key_hr = {}
+                    for key, value in r_key.items():
+                        if value or value is False:
+                            r_key_hr[capitalize_first_letter(key)] = value
+                    registry_keys_hr.append(r_key_hr)
+                registry_keys_title = 'Registry Keys for Alert'
+                hr += tableToMarkdown(registry_keys_title, registry_keys_hr, removeNull=True)
+        context = {
+            'ID': alert_details['id'],
+            'Title': alert_details['title'],
+            'Category': alert_details['category'],
+            'Severity': alert_details['severity'],
+            'CreatedDate': alert_details['createdDateTime'],
+            'EventDate': alert_details['eventDateTime'],
+            'Status': alert_details['status'],
+            'Vendor': alert_details['vendorInformation']['vendor'],
+            'Provider': alert_details['vendorInformation']['provider']
+        }
+        ec = {
+            'MsGraph.Alert(val.ID && val.ID === obj.ID)': context
+        }
     return hr, ec, alert_details
 
 
 def update_alert_command(client: MsGraphClient, args):
     alert_id = args.get('alert_id')
-    vendor_information = args.get('vendor_information')
+    status: str = args.get('status', "")
+    if status == "newAlert" and API_VER == API_V2:
```

Comment:
is this a possibility now, Didnt we take this out of the yml?
Could this be a problem on the flip side? Being "new" while using v1?

Reply:
We talked about it in Slack.
We decided to keep newAlert and remove new (change to new in that case) to avoid breaking bc.

Reply:
oh newAlert is the old 1?